### PR TITLE
fix TestEVHUpdateIngressClassWithInfraSetting UT

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by akshayhavile** [comment]
Can one of the admins verify this patch?

**Comment by swathinsankaran** [comment]
build ako

**Review by swathinsankaran — DISMISSED** [review]
LGTM

File: tests/evhtests/l7_evh_ingressclass_test.go [hunk 1]
```diff
@@ -652,6 +652,11 @@ func TestEVHUpdateIngressClassWithInfraSetting(t *testing.T) {
 		found, _ := objects.SharedAviGraphLister().Get(settingModelName1)
 		return found
 	}, 40*time.Second).Should(gomega.Equal(true))
+	g.Eventually(func() int {
+		_, aviSettingModel := objects.SharedAviGraphLister().Get(settingModelName1)
+		settingNodes1 := aviSettingModel.(*avinodes.AviObjectGraph).GetAviEvhVS()
+		return len(settingNodes1[0].EvhNodes)
+	}, 40*time.Second).Should(gomega.Equal(2))
 	_, aviSettingModel1 := objects.SharedAviGraphLister().Get(settingModelName1)
```
Comment (akshayhavile) [inline tests/evhtests/l7_evh_ingressclass_test.go:660]: @arihantg: Should we take out line 660 and 661?

Comment (arihantg) [reply tests/evhtests/l7_evh_ingressclass_test.go]: addressed.

**Comment by arihantg** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**